internal/resources/ekscluster/ekscluster_flatten_test.go

```diff
@@ -73,6 +208,30 @@ func TestFlattenCluterSpec(t *testing.T) {
 									},
 								},
 							},
+							"addons_config": []interface{}{
+								map[string]interface{}{
```

Comment:
Lets add a test for empty security group, and we need to make sure if an empty security group is specified, the TF state file has the cluster security group set after cluster creation.

Reply:
@DheerajSShetty  the data comes back from API and adding a empty security group would not serve the purpose as the expectation is for the API to return a security group back from the cluster though it was not added in the eniconfig. This is in line with our discussion. I have manually tested the terraform file with empty security groups under the eni_config and it still sets the cluster security group under eni_config as expected in the state file.